Documentation/doc/Documentation/Doxyfile.in

```diff
@@ -1,6 +1,6 @@
 # this package is a little peculiar and different from the other packages
 @INCLUDE = ${CGAL_DOC_PACKAGE_DEFAULTS}
-PROJECT_NAME = "CGAL ${CGAL_DOC_VERSION} - Manual"
+PROJECT_NAME = "C&zwj;GAL ${CGAL_DOC_VERSION} - Manual"
```

Comment:
I am against that patch. The [zero-width joiner](https://en.wikipedia.org/wiki/Zero-width_joiner) is not made for that sort of thing. "C ZWJ G" should display one letter that is the union of C and G in one letter.

Maybe the [zero-width non-joiner](https://en.wikipedia.org/wiki/Zero-width_non-joiner) could be used, but that is a very nasty visual trick: "C ZWNJ G" should display a ligature of C and G. There might be fonts were that ligature exist.

Reply:
Even with the [zero-width space](https://en.wikipedia.org/wiki/Zero-width_space) the effect will be to break CGAL into two words: "C" and "GAL", but joined by a zero-width space. Even that is a nasty visual trick. Why this workaround, by the way?